target/linux/ath79/image/generic.mk

```diff
@@ -1160,6 +1160,24 @@ define Device/meraki_mr16
 endef
 TARGET_DEVICES += meraki_mr16
 
+# TODO
+# Refactor MR16 / MR12 around single device
```

Comment:
Yes, just create a meraki_mr or similar for the shared parts.

Reply:
Done. I used meraki_mr_ath to avoid conflicts as best as possible.